### DISCUSSION TITLE
Call nearby_charging_sites Endpoint

### DISCUSSION THREAD

**Discussion by jheredianet** [tobiasehlert/teslamateapi#263] [General]
Hi, I'm just wondering if there is a way to call: **nearby_charging_sites**

When I call _/api/v1/cars/1/command/nearby_charging_sites_ (POST call), I get:

{
  "error": "unauthorized"
}

I see with this: _/api/v1/cars/1/command_ the available commands and see that this is not implemented, as well as others. 
I'm just curious is there is a wall to call them. 

Thanks.

**Comment by tobiasehlert** [comment]
hi @jheredianet,

As of today we do not have support for [state](https://tesla-api.timdorr.com/vehicle/state) API endpoints implemented, just the [command](https://tesla-api.timdorr.com/vehicle/commands) API endpoints. Therefore the request is not forwarded as intended from you.

I know that @LelandSindt is looking into the upcoming changes of the Tesla API in #254 so that might something you want to follow as well.

Regards,
Tobias

**Reply by jheredianet** [reply]
Thanks @tobiasehlert , I'll keep an eye out for any news.
